TELCODOCS#1877: Enabling the SR-IOV network metrics exporter (#80754)
openshift/openshift-docs: modules/nw-sriov-configuring-operator.adoc

  enableInjector: true
  enableOperatorWebhook: true
  logLevel: 2

>   featureGates: false

Author: https://80754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-operator.html#sriov-operator-metrics_configuring-sriov-operator

Set to `0` to show only the basic logs. Set to `2` to show all the available logs.
By default, this field is set to `2`.

> |`spec.featureGates`
> |`boolean`
> |Specifies whether to enable or disable the optional features.
> By default, this field is set to `false`.
> 

Reviewer: ```suggestion
|`spec.featureGates`
|`map[string]bool`
|Specifies whether to enable or disable the optional features, e.g. `metricsExporter.
```

Author: Updated as per the suggestion.
